src/evaluator.ts

```diff
@@ -448,6 +454,12 @@ export class Evaluator {
       callback,
     );
     if (variation) {
+      debugEvalSuccess(
+        `${JSON.stringify(variation.value)}`,
```

Comment:
Won't this already be a JSON encoded string?

Reply:
In my real testing, I confirmed it was an `Object`
![Screenshot 2023-06-23 at 12 42 53](https://github.com/harness/ff-nodejs-server-sdk/assets/23323369/4fb6d8b3-ed68-4661-bb18-c68df7c6801c)
t`

Reply:
If that's the case then line 463 should explode when you try to `JSON.parse` an object
<img width="452" alt="Screenshot 2023-06-23 at 12 49 37" src="https://github.com/harness/ff-nodejs-server-sdk/assets/7167253/aa693f19-16f6-41f6-b257-6d7d350c94b9">

Reply:
Blooming heck you're right it is a JSON string already. Sorry. Fixed